### ISSUE TITLE
onChange function not working.

### ISSUE THREAD

**Issue by Soundvessel** [pvoznyuk/react-live-clock#25]
Tried the following implementation with version 3.1.0 and I am getting no console log from onChange function.

```
import React from "react"
import ReactDOM from "react-dom"
import Clock from "react-live-clock"

import "./styles.css"

console.log("test")

function App() {
  return (
    <div className="App">
      <Clock
        format={"HH:mm:ss"}
        ticking={true}
        onChange={() => console.log("tick")}
      />
    </div>
  )
}

const rootElement = document.getElementById("root")
ReactDOM.render(<App />, rootElement)
```

Nothing in console. You can view and modify in codesandbox here: https://codesandbox.io/s/880zo5v9ml

**Comment by mrkerr** [comment]
I'm having this issue as well

**Comment by anthony0030** [comment]
We have just published version 4.0.0 it is still in beta as I am still testing it. 
Could you please check to see if you are having this problem with version:
"react-live-clock": "^4.0.0-beta" 

It has a minimum react version "react": "^16.8" because it makes usage of react hooks.

**Comment by Soundvessel** [comment]
Reran the above code sandbox with the updated version and it works.
